### ISSUE TITLE
error on tree count here at magic trees priff world 444 for myself

### ISSUE THREAD

**Issue by Echo-Hollow** [Infinitay/tree-count-plugin#24]
![issue](https://github.com/Infinitay/tree-count-plugin/assets/133886621/1c3c13f7-2815-4354-85cc-ea1d386f9860)

**Comment by Infinitay** [comment]
Is it happening consistently? Also, could you give me a broader screenshot so I know exactly where it is? I don't have members at the moment, but I'll try and work something out

**Comment by Echo-Hollow** [comment]
Is it happening consistently?

Yes.

Also, could you give me a broader screenshot so I know exactly where it is?

I am not on my device atm, but it is one of the prif (elf) resource area,
near the altar/bank/agility-course.

 I don't have members at the moment, but I'll try and work something out

Much appreciated. Thanks for the work you do sir.

**Comment by Infinitay** [comment]
I finally got membership and back on OSRS, so I'll look into this today. Hopefully I can work this out by today.

**Comment by Infinitay** [comment]
The problem seems to be with felling axes. I only tested this with the dragon felling axe, but I assume that all felling axes have the same behavior. Unlike when chopping with regular axe such as the dragon axe counter-part, it seems that the game doesn't re-orient the player to the respective tree when chopping a tree. This leaves the player chopping into the air until they manually click on the tree again.


https://github.com/Infinitay/tree-count-plugin/assets/6964154/3a46f93b-ef7e-4370-bea3-0c87bdf43ef6

I could push a fix that will check to see if a player is chopping with a felling axe and they are next to one tree, but this wouldn't work when there are close trees nearby such as here in Prif or in the woodcutting guild (not that the plugin works there anyways since the hidden boost isn't available there). Although currently I am opposed to doing this because it won't be 100% accurate when there are multiple trees _0 tiles away_

![image](https://github.com/Infinitay/tree-count-plugin/assets/6964154/e93efd5b-ca88-4e07-939c-3f063128ac0f)

**Comment by Echo-Hollow** [comment]
I am unsure if the answer for this request. I think I'll have a look on the
forums for similar issues if people can her or not etc. I'll see too !

**Comment by Infinitay** [comment]
I fixed some logic bugs and also made a temporary workaround to another issue that is caused due to an incorrect woodcutting animation. Unfortunately fixing the animation is not in my control, but I've tried to make it as accurate as I could while not making too many assumptions. We'll have to wait for the RuneLite team to accept the plugin changes and push the update to everyone.

https://github.com/runelite/plugin-hub/pull/5224

I'll leave the issue open for a while in case anyone is still experiencing problems.
